Documentation/README_Interactable.md

```diff
@@ -0,0 +1,285 @@
+# Interactable
+![Interactable](/External/ReadMeImages/Interactable/InteractableExamples.png)
+
+## How to use Interactable
+Add the Interactable Component to a GameObject
+
+![Interactable](/External/ReadMeImages/Interactable/InteractableInspector_basicSteps.png)
+
+1. A Collider must exist on the GameObject with the Interactable or the child of the Interactable for it to receive input.
+2. Use the OnClick event to make something happen.
+3. Add visual feedback by linking a Target to a Profile and assigning a Theme.
+
+
+## Input Settings
+The basic features allow for button style interactions, such as pointer focus and clicks, that maps to interaction states to drive themes which are setup through the Interactable profile. Controller or hand focus, down, up and click (both near and far) are handled. Functionality can be extended using external scripts that can set state manually.
+
+<img src="/External/ReadMeImages/Interactable/InputFeatures_short.png" width="450">
+
+### Input Actions
+Select the action, from the input configuration or controller mapping profile, that the Interactable should react to.
+See the InputProfile or the DefaultMixedRealityInputActionsProfile for more on how Input Actions are setup and intended to be used in the application.
+ 
+### Enabled
+Sets the Interactable's enabled state, which will disable some input handling and update the themes to reflect the current state which is disabled.
```

Comment:
I think it actually may be helpful here to have the example first (i.e. visible submit button that looks greyed out) - the first part of this paragraph was slightly confusing to me to read until I got the specific example (i.e. there is an overloading of the term "enabled/disabled" which comes from maybe flat UI interaction land vs. the concept of enabled/disabled in Unity land)

I otherwise would have changed this myself but I wanted to see your thoughts on this.

Reply:
@killerantz Could you please take a look?

Reply:
I can see the confusion, maybe we do have an example of a button in four different states to show the visual differences and try to bring the reference back to UI states instead of Unity functionality.

Reply:
I'll push up a couple more images tomorrow.